src/views/credits/people.json

```diff
@@ -53,6 +58,11 @@
         "userName": "codubee",
         "userId": 10866958,
         "name": "Colby"
+    },
+     {
+        "userName": "noncanonical",
+        "userId": 55851826,
+        "name": "Craig"
     },
     {
         "userName": "harakou",
```

Comment:
It should be Harakou. But it is simple though.

Reply:
Got it! I fixed it!

Reply:
Great! It was not a bug produced by you though.

Reply:
Oh well.